core/src/main/java/hudson/slaves/RetentionStrategy.java

```diff
@@ -217,13 +238,38 @@ public long getIdleDelay() {
             return idleDelay;
         }
 
+        /**
+         * Getter for property 'conflictsWith'.
+         *
+         * @return Value for property 'conflictsWith'.
+         */
+        public String getConflictsWith() {
```

Comment:
add a `doCheckConflictsWith` method to validate the regex?

Reply:
fixed